BrainPortal/lib/cbrain_task_generators/schema_task_generator.rb

```diff
@@ -166,6 +166,20 @@ def integrate(register: true, create_tool_config: false, multi_version: false)
         })[partial]
       end
 
+      # Write out a help file for this Boutiques task
+      helpFileName = name + "_help.html" 
+      helpFileDir = "cbrain_plugins/cbrain_tasks/help_files/"
+      basePath = Rails.root.join('public/' + helpFileDir) 
+      FileUtils.mkdir_p basePath.to_s # creates directory if needed
```

Comment:
There is a (tacit?) convention in CBRAIN that we use parentheses in method arguments.